### PR TITLE
geany.desktop.in: Add Keywords field entry

### PR DESCRIPTION
**Pull request by erikd**
This entry matches the `Keywords` field in Gedit.

### CODE REVIEW

File: geany.desktop.in [hunk 1]
```diff
@@ -10,3 +10,4 @@ Terminal=false
 Categories=GTK;Development;IDE;
 MimeType=text/plain;text/x-chdr;text/x-csrc;text/x-c++hdr;text/x-c++src;text/x-java;text/x-dsrc;text/x-pascal;text/x-perl;text/x-python;application/x-php;application/x-httpd-php3;application/x-httpd-php4;application/x-httpd-php5;application/xml;text/html;text/css;text/x-sql;text/x-diff;
 StartupNotify=true
+Keywords=Text;Editor;
```
Comment (b4n) [inline geany.desktop.in]: Should be translatable (prefixed with a `_`)

https://specifications.freedesktop.org/desktop-entry-spec/latest/ar01s05.html
https://git.gnome.org/browse/gedit/tree/data/org.gnome.gedit.desktop.in.in#n19

**Comment by erikd** [comment]
Updated.

**Comment by b4n** [comment]
LGBI, might require checking actual i18n

**Comment by frlan** [comment]
Will check for 18n and merge after in case of its working.

**Pull request merged**